stable/ibex-dashboard/templates/storageclass.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClassName) -}}
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+ name: "{{ .Values.persistence.storageClass }}"
+provisioner: {{ .Values.persistence.storage.provisioner }}
+parameters:
```

Comment:
Question about cloud provider assumption: is Ibex Dashboard only applicable to run on Azure? I wasn't able to determine that just by reading the [project README](https://github.com/CatalystCode/ibex-dashboard/blob/master/README.md), and I don't have a setup on Azure to get familiar with the application.

I ask because the template as constructed (`skuName` etc) appears that it would only succeed with `kubernetes.io/azure-disk` or `kubernetes.io/azure-file`.

Reply:
This runs perfectly well with any setup you choose and any pvc.
It works well on local container deployment.
The default values demonstrate how to configure it, but any other settings will be fine.